package "result" 1.0 fails to build

I get the following error when trying to install "result" (1.0) on both OS X and Linux:

=-=- Processing actions -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
[ERROR] File "$HOME/.opam/4.02.3/build/result.1.0/result.install", line 1, character 0-1: '-' is not a valid token.

#=== ERROR while installing result.1.0 ========================================#
OpamTypesBase.Lexer_error("'-' is not a valid token")

--- a/packages/result/result.1.0/opam
+++ b/packages/result/result.1.0/opam
@@ -6,3 +6,4 @@ dev-repo: "https://github.com/janestreet/result.git"
 bug-reports: "https://github.com/janestreet/result/issues"
 license: "BSD3"
 build: [[make]]
+available: [os = "linux"]

--- a/packages/result/result.1.1/descr
+++ b/packages/result/result.1.1/descr
@@ -1,0 +1,5 @@
+Compatibility Result module
+
+Projects that want to use the new result type defined in OCaml >= 4.03
+while staying compatible with older version of OCaml should use the
+Result module defined in this library.

--- a/packages/result/result.1.1/opam
+++ b/packages/result/result.1.1/opam
@@ -1,0 +1,8 @@
+opam-version: "1.2"
+maintainer: "opensource@janestreet.com"
+authors: ["Jane Street Group, LLC <opensource@janestreet.com>"]
+homepage: "https://github.com/janestreet/result"
+dev-repo: "https://github.com/janestreet/result.git"
+bug-reports: "https://github.com/janestreet/result/issues"
+license: "BSD3"
+build: [[make]]

--- a/packages/result/result.1.1/url
+++ b/packages/result/result.1.1/url
@@ -1,0 +1,2 @@
+archive: "https://github.com/janestreet/result/archive/1.1.tar.gz"
+checksum: "1e0005783d4f70e1a9772723fa0e846b"

